### ISSUE TITLE
Change link in repository description

### ISSUE THREAD

**Issue by michalbundyra** [zendframework/zend-mvc#264]
Please change link in the repository description:
currently we have: https://github.com/zendframework/zf2
but IMHO we should have there: https://docs.zendframework.com/zend-mvc/

It would be nice also add some tags to the repo. Maybe the same as we have in composer.json?

**Comment by froschdesign** [comment]
We should remove the link from the repository description.

**Comment by Xerkus** [comment]
@froschdesign offtopic: can't find you in slack. Are you registered there?

**Comment by michalbundyra** [comment]
@froschdesign 
> We should remove the link from the repository description.

I disagree, in other repositories we have it set (maybe I wasn't clear, it's not in description but in separate field "website") - and also "sync-repos" script in maintainers repository updates website to be link to the docs

https://github.com/zendframework/maintainers/blob/332fb4386235fe1de8aec71aa014ccbdd32a412b/src/SyncRepos.php#L53

**Comment by froschdesign** [comment]
@webimpress 
> maybe I wasn't clear, it's not in description but in separate field "website"

You are right, there are two fields – description and website.

**Comment by Xerkus** [comment]
@weierophinney can you update the repo description please?

**Comment by weierophinney** [comment]
This repository has been closed and moved to laminas/laminas-mvc; a new issue has been opened at https://github.com/laminas/laminas-mvc/issues/16.
